python/GafferSceneUI/RenderPassEditor.py

```diff
@@ -67,6 +68,8 @@ def __init__( self ) :
 			self["editScope"] = Gaffer.Plug()
 			self["displayGrouped"] = Gaffer.BoolPlug()
 
+			self["favouriteColumns"] = Gaffer.StringVectorDataPlug()
```

Comment:
I wonder what the plug layout would look like if we extend this to support multiple favourites tabs, or to control visibility of columns in the default tabs. Perhaps we don't need to worry too much because this is a UI class, and serialisation backward compatibility isn't an issue?